Libraries/RoslynTripleSlash/TripleSlashSyntaxRewriter.cs

```diff
@@ -0,0 +1,409 @@
+            // No need to add exceptions in secondary files
+            if (!UseBoilerplate && member.Exceptions.Any())
+            {
+                foreach (SyntaxTriviaList exceptionsTrivia in member.Exceptions.Select(
+                    exception => GetException(exception.Cref, exception.Value, leadingWhitespace)))
+                {
+                    exceptions = exceptions.AddRange(exceptionsTrivia);
+                }
+            }
+            return exceptions;
+        }
+
+        private SyntaxTriviaList GetException(string cref, string text, SyntaxTriviaList leadingWhitespace)
+        {
+            TypeCrefSyntax crefSyntax = SyntaxFactory.TypeCref(SyntaxFactory.ParseTypeName(cref));
+            XmlTextSyntax contents = SyntaxFactory.XmlText(GetTextAsTokens(text, leadingWhitespace));
+            XmlElementSyntax element = SyntaxFactory.XmlExceptionElement(crefSyntax, contents);
+            return GetXmlTrivia(element, leadingWhitespace);
+        }
+
+        private SyntaxTriviaList GetSeeAlsos(DocsMember member, SyntaxTriviaList leadingWhitespace)
+        {
+            SyntaxTriviaList seealsos = new();
+            // No need to add exceptions in secondary files
+            if (!UseBoilerplate && member.SeeAlsos.Any())
+            {
+                foreach (SyntaxTriviaList seealsoTrivia in member.SeeAlsos.Select(
+                    s => GetSeeAlso(s.Cref, leadingWhitespace)))
+                {
+                    seealsos = seealsos.AddRange(seealsoTrivia);
+                }
+            }
+            return seealsos;
+        }
+
+        private SyntaxTriviaList GetSeeAlso(string cref, SyntaxTriviaList leadingWhitespace)
+        {
+            TypeCrefSyntax crefSyntax = SyntaxFactory.TypeCref(SyntaxFactory.ParseTypeName(cref));
+            XmlEmptyElementSyntax element = SyntaxFactory.XmlSeeAlsoElement(crefSyntax);
+            return GetXmlTrivia(element, leadingWhitespace);
+        }
+
+        private SyntaxTokenList GetTextAsTokens(string text, SyntaxTriviaList leadingWhitespace)
+        {
+            string whitespace = leadingWhitespace.ToFullString().Replace(Environment.NewLine, "");
+            var tokens = new List<SyntaxToken>();
+            foreach (string line in text.Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries))
+            {
+                tokens.Add(SyntaxFactory.XmlTextNewLine(Environment.NewLine + whitespace)); // Needs to be textnewline, and below needs to be textliteral
+                tokens.Add(SyntaxFactory.XmlTextLiteral(SyntaxTriviaList.Create(SyntaxFactory.SyntaxTrivia(SyntaxKind.DocumentationCommentExteriorTrivia, string.Empty)), line, line, default));
+            }
+
+            return SyntaxFactory.TokenList(tokens);
+        }
+
+        private SyntaxList<XmlNodeSyntax> GetContentsInRows(string text)
+        {
+            return new(SyntaxFactory.XmlText(text)); // TODO: Press enter!
+        }
```

Comment:
This should not be needed anymore. I solved it with the for loop in line 337.

Reply:
Actually, it's not exactly the same. This one is a list of `XmlNodeSyntax`, that one is a list of `SyntaxToken`s. But the string splitting needs to be done the same way.